Automattic/wp-calypso
# Social signup: Show loading while sending magic link email

Related to #83871

## Proposed Changes

* Add visual loading while sending the email, this prevents the render of two different components and the “jump feeling”. More details https://github.com/Automattic/wp-calypso/issues/83871

## Testing Instructions



* Use live link
* Navigate to /log-in on incognito
* Use a passwordless email and click on continue
* You should see the loading indicator then a message "Check your email"
* If you use a sus email, you should see the send link view "Email me a login link"


## Pre-merge Checklist
